src/tags/tests/test_models.py

```diff
@@ -104,6 +104,61 @@ def test_multiple_badge_assertion_tagged(self):
         badge_assertion_qs = get_badge_assertion_by_tags(self.user, ['tag0', 'tag1', 'tag2'])
         self.assertEqual(badge_assertion_qs.count(), 6)
 
+    def test_correct_quest_submission_tagged(self):
+        """ check if 'correct tag' is retrieved and no other submission from a different tag or no tag """
+        # with correct tag
+        co_1, cs_1 = self.create_quest_and_submissions(1)
```

Comment:
don't use variables names like this.  use informative full words, sentences if you have to.  Reading this code, I don't know what data those variables are holding.  what's a co and a ca?

Reply:
fixed it